docs/backend.md

```diff
@@ -4,9 +4,30 @@ This site is built using Drupal and combination of modules and themes.
 ## Drupal
 As of the writing of this document the foundation of the site is built using Drupal + Symphony + Composer. Drupal documentation can be found here: https://www.drupal.org/docs. This site follows Drupal coding standards and best practices.
 
+### Updating Drupal via Composer
+Drupal projects managed through Composer typically use `drupal/core-recommended`, which ensures that Drupal core and its dependencies are installed with secure and tested versions.
+
+For ongoing updates we use the following command `composer update drupal/core-* -W` which will take care of both updating Drupal core any any dependencies that it may require. This will result in an updated `composer.lock` file which will be committed to the repository once the update has been tested locally.
+
+Once updated, the following drush command should be issued, `drush updatedb && drush cr`. This will check for any pending database updates followed up by a clearing of the cache.
+
 ## Modules
 Modules are located in `/web/modules/` directory. Each module has an module page located on https://www.drupal.org/project/project_module. The modules page include documentation, versions, change details and an issue queue. Also most modules contain additional details in a README file in the module root.
 
+### Installing modules via Composer
+To add modules, use the Composer `require` command. For example, to add the **Admin Toolbar** module:
+
+`composer require drupal/admin_toolbar`
+
+This will download the module and ensure it is included in your `composer.json` file, allowing it to be version-controlled and managed by Composer. Additionally this will update your `composer.lock` file which will reflect the details of installed packages and dependencies as it relates to the module being installed. These changes will, again, require you to commit the files to the repository once testing has been done locally.
+
+### Updating modules via Composer
+To update a specific module, for example, **Admin Toolbar**:
+
+`composer update drupal/admin_toolbar`
+
+This ensures that both the module and its dependencies are updated according to Drupal's compatibility guidelines.
```

Comment:
We should also include an updb step here and drush cex as well to make sure we capture any changes to config.